admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -156,19 +156,66 @@ that cached objects are available for as long as they are needed.
 
 The Redis PHP module must be version 2.2.5 and up.
 
-On Debian/Ubuntu/Mint install ``redis-server`` and ``php5-redis``. The installer 
-will automatically launch ``redis-server`` and configure it to launch at 
-startup.
-
-On Red Hat/CentOS/Fedora install ``redis`` and ``php-pecl-redis``. It will not 
-start automatically, so you must use your service manager to start 
-``redis``, and to launch it at boot as a daemon.
+- On Debian/Ubuntu/Mint install Redis with  ``apt-get install redis-server`` and 
+  ``apt-get install php5-redis``. The installer will automatically launch 
+  ``redis-server`` and configure it to launch at startup.
+
+.. note:: On Ubuntu 14.04 (Trusty), the ``php5-redis`` version is too old and not supported.
+  You can check what version you have installed with:
+  
+  .. code-block:: bash
+  
+   apt-show-versions php5-redis
+   php5-redis:amd64/trusty 2.2.4-1build2 uptodate
+  
+  | This example shows an outdated version. 
+  | In case you have this version, you must uninstall this version with: 
+  
+  .. code-block:: bash
+  
+   apt-get remove php5-redis
+
+  and install a newer version of Redis. 
+  
+  One solution is to look for a PPA. See `launchpad <https://launchpad.net/ubuntu/+source/php-redis>`_.
+  
+  Another and preferred solution option is using the PECL/PEAR PHP extension repository:
```

Comment:
I don't think this should be the preferred solution as you won't get updates if installed manually. I also don't know if this can cause troubles if the system is upgraded to a newer release like 15.04/15.10/16.04